### PR TITLE
Update Inventory REST tutorial to include in-store pickup

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds a step to the Inventory REST tutorial that demonstrates the in-store pickup feature. To do this, I had to update the source examples. I also moved the "Retrieve the order item IDs" section from step 9 to step 10, because it was in the wrong place.

## Affected DevDocs pages

-  guides/v2.4/rest/tutorials/inventory/in-store-pickup.html

## Links to Magento source code

-  ...

whatsnew
Expanded the [Order processing with Inventory Management](https://devdocs.magento.com/guides/v2.4/rest/tutorials/inventory/index.html) REST tutorial to include a new step that demonstrates the [in-store pickup](https://devdocs.magento.com/guides/v2.4/rest/tutorials/inventory/in-store-pickup.html) feature.

### CODE REVIEW

File: src/guides/v2.4/rest/tutorials/inventory/in-store-pickup.md [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+menu_order: 140
+level3_subgroup: msi-tutorial
+return_to:
+  title: REST Tutorials
+  url: rest/tutorials/index.html
+functional_areas:
+  - Integration
+---
+
+The in-store pickup feature allows shoppers to select a physical location wear they can pick up an entire order. As a result, any orders that are fulfilled with this method bypass the shipping steps in the standard workflow.
```
Comment (jeff-matthews) [inline src/guides/v2.4/rest/tutorials/inventory/in-store-pickup.md:16]:
```suggestion
The in-store pickup feature allows shoppers to select a physical location where they can pick up an entire order. As a result, any orders that are fulfilled with this method bypass the shipping steps in the standard workflow.
```

File: src/guides/v2.4/rest/tutorials/inventory/in-store-pickup.md [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+}
+```
+
+## Set the shipping and billing information
+
+The customer has chosen to pick up the order at the Brooklyn store. Once again, we will use the `POST /V1/carts/mine/shipping-information` endpoint to set the shipping and billing information. This time we will make the following changes to the payload:
+
+*  The shipping address is that of the Brooklyn store but includes the customer's first and last name to help the store personnel identify who placed the order. The shipping address also includes the `pickup_location_code` extension attribute.
+
+*  The billing address includes ID of the customer's billing address. In [Step 6. Create a customer and generate a customer token]({{page.baseurl}}/rest/tutorials/inventory/create-customer.html), the `POST /V1/customers` endpoint returned an address ID of `2`.
```
Comment (jeff-matthews) [inline src/guides/v2.4/rest/tutorials/inventory/in-store-pickup.md:202]:
```suggestion
*  The billing address includes the customer's billing address ID. In [Step 6. Create a customer and generate a customer token]({{page.baseurl}}/rest/tutorials/inventory/create-customer.html), the `POST /V1/customers` endpoint returned an address ID of `2`.
```

**Review by dmytro-ch — APPROVED** [review]
From a technical perspective, the changes look good to me. ✔️ 
Thank you!

**Comment by keharper** [comment]
running tests

**Pull request merged**